lib/web_ui/lib/src/engine/window.dart

```diff
@@ -182,6 +172,9 @@ class EngineWindow extends ui.Window {
         textEditing.channel.handleTextInput(data);
         return;
 
+      case 'flutter/web_test_e2e':
+        _handleWebTestEnd2EndMessage(data);
+        return;
```

Comment:
Does testing with mobile emulation ok? Looks like there is an option for setting the device pixel ratio for chrome driver:http://chromedriver.chromium.org/mobile-emulation 
Currently we don't have a way of passing options to the 'flutter drive' command. 

Can you please add a todo? I can remove this part once I added options to chrome driver :)

Reply:
We would need to be able to pass all other browsers as well for integration tests. There are issues floating around from 2018 still not fixed. We may have to keep this for some browsers for a while. For Chrome we could use option, for other browsers use this channel.

Reply:
Thanks for the clarification.